### PR TITLE
Fix regex

### PR DESCRIPTION
**Pull request by plynchnlm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +25,7 @@ This log documents significant changes for each release.  This project follows
 ## [0.10.2] - 2019-02-26
 ### Fixed
  - Corrected the behavior of xor with an empty set.
+>>>>>>> master
```
Comment (xluanlhc) [inline CHANGELOG.md]: Clean up here

File: src/fhirpath.js [hunk 2]
```diff
@@ -158,7 +158,32 @@ engine.LiteralTerm = function(ctx, parentData, node) {
 };
 
 engine.StringLiteral = function(ctx, parentData, node) {
-  return [node.text.replace(/(^['"]|['"]$)/g, "")];
+  // Remote the beginning and ending quotes.
```
Comment (xluanlhc) [inline src/fhirpath.js]: Remove (typo)

File: CHANGELOG.md (hunk 1 above)
Comment (plynchnlm) [reply CHANGELOG.md]: Deleted

File: src/fhirpath.js (hunk 2 above)
Comment (plynchnlm) [reply src/fhirpath.js]: Fixed.

**Pull request merged**